keyboards/aplusx_pulsar/xboard_qs/keymaps/via/keymap.c

```diff
@@ -0,0 +1,371 @@
+/* Copyright 2024 Aplusx Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+static bool     INIT3S_on = false;
+static bool     INIT3S_led_on = false;
+static uint16_t INIT3S_timer;           // for custom key, INIT_3S
+
+static bool     kvm_pc_sel      = false;
+static uint16_t kvm_timer;
+static bool     kvm_sel_on      = false;
+
+static bool isMacMode = false;
+
+static uint8_t Status_LED_Bright;       // maybe caps, scroll, PC1/2, win/mac, dynamic macro, win lock
+                                           // 0 = 초기값, 1~5 = 50~250의 5단계 밝기
+#define kvm_deadtime    100
+//------------------------------
+typedef union {
+    uint32_t raw;
+    struct {
+        bool    eeprom_kvm_pc_sel :1;
+        uint8_t eeprom_Status_LED_Bright;       //
+    };
+} user_config_t;
+
+user_config_t user_config;
+
+//-=-------------------------------------------
+    enum _layer { WIN_BASE = 0, WIN_FN, MAC_BASE, MAC_FN };
+
+    enum pulsar_keycodes {
+        KC_TGUI = QK_KB_0,  // Ver 0.21.6에서 변경 (USER00,)  // Toggle between GUI Lock or Unlock
```

Comment:
This is a keyboard keycode, not a keymap/user keycode.
```suggestion
        KC_TGUI = QK_USER_0,  // Ver 0.21.6에서 변경 (USER00,)  // Toggle between GUI Lock or Unlock
```

Reply:
Thank you. Well understood